src/index.test.js

```diff
@@ -199,7 +199,7 @@ describe('#jsx-pdf', () => {
           <content>
             Hello{null}
             {undefined}
-            {''}
+            {'' /* eslint-disable-line react/jsx-curly-brace-presence */}
```

Comment:
We still want to test this, so it makes sense to disable the rule for this line.